sdk/src/main/java/com/silanis/esl/sdk/examples/AccountRolesExample.java

```diff
@@ -0,0 +1,51 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.sdk.builder.AccountRoleBuilder;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public class AccountRolesExample extends SDKSample {
+    List<com.silanis.esl.sdk.AccountRole> result = null;
+    List<com.silanis.esl.sdk.AccountRole> result2 = null;
+    List<com.silanis.esl.sdk.AccountRole> result3 = null;
+    String newAccountRoleId = null;
+    com.silanis.esl.sdk.AccountRole newAccountRole = null;
+    List<String> newAccountUsers = null;
+
+    public static void main(String... args) {
+        new AccountRolesExample().run();
+    }
+
+    public void execute() {
+        result = eslClient.getAccountService().getAccountRoles();
+        String newAccountRoleName = UUID.randomUUID().toString();
+        com.silanis.esl.sdk.AccountRole accountRole = AccountRoleBuilder.newAccountRole()
+            .withName(newAccountRoleName)
+            .withPermissions(Collections.singletonList(UUID.randomUUID().toString()))
+            .withDescription("DESCRIPTION")
+            .withEnabled(true)
+            .build();
+
+        eslClient.getAccountService().addAccountRole(accountRole);
+
+        for (com.silanis.esl.sdk.AccountRole forAccountRole : result2) {
+            if (forAccountRole.getName().equals(newAccountRoleName)) {
```

Comment:
Isn't there any possibility of null pointer exception?

Reply:
It's an example, garbage in, garbage out. If the person push a role without a defined name, he might get a null in it's name. But in my example the name is provided, so it can't be NULL.

Reply:
Then I have two questions.
1. So if you are defining AccountRoleName with random value rather than a specific value, is it because you cannot have the same AccountRoleName in one account?
2. In this code, result2 is null, so why do you think this loop will be executed?

Reply:
I fixed the example